data/themes/default/style.css

```diff
@@ -114,9 +113,15 @@ PianoRoll {
 	qproperty-gridColor: rgb( 128, 128, 128 );
 	qproperty-noteModeColor: rgb( 255, 255, 255 );
 	qproperty-noteColor: rgb( 119, 199, 216 );
-	qproperty-barColor: #4afd85;
 	qproperty-noteBorderRadiusX: 5;
 	qproperty-noteBorderRadiusY: 2;
+	qproperty-selectedNoteColor: rgb( 0, 64, 192 );
+	qproperty-barColor: #4afd85;
```

Comment:
>@Umcaruje food for thought... https://github.com/yann-lty/qtsass

That would be awesome to reduce the redundancy in the source code and would allow for easier theming, but its written in python, and I don't know if its worth adding such a dependency just for CSS. 

> then perhaps a cmake version of http://lesscss.org/. (Wishful thinking? :smile_cat: )

LESS/SASS can't work with Qt directly, because QtCSS has some differences from normal CSS and they're not compatible, hence the `qtsass` project. I couldn't find a Less to QtCSS converter.